src/lib/rest/httpRequestSend.cpp

```diff
@@ -212,6 +213,28 @@ static int contentLenParse(char* s)
 
 /* ****************************************************************************
 *
+* servicePathForMetrics - extract first component of service-path
+*/
+static void servicePathForMetrics(const char* servicePath, char* servicePath0, int servicePath0Len)
```

Comment:
Discussed at skype.

* New name: firstServicePath
* Move to the library which deals with other service path manitpulations, e.g. spliting service path header string into vector. It seems that library is rest.cpp/h